composer.json

```
@@ -9,20 +9,20 @@
     "docs": "https://googleapis.github.io/google-auth-library-php/main/"
   },
   "require": {
-    "php": "^7.1||^8.0",
-    "firebase/php-jwt": "^5.5||^6.0",
+    "php": "^7.4||^8.0",
+    "firebase/php-jwt": "^6.0",
     "guzzlehttp/guzzle": "^6.2.1|^7.0",
-    "guzzlehttp/psr7": "^1.7|^2.0",
+    "guzzlehttp/psr7": "^2.0",
```

Seems like older guzzle versions suffer from security vulnerabilities: https://packagist.org/packages/guzzlehttp/psr7/advisories?version=7023692
```suggestion
    "guzzlehttp/psr7": "^2.4.5",
```

---

good catch! updated.